src/pages/_app.js

```diff
@@ -9,6 +9,7 @@ import { MembersProvider } from '@store';
 import { UserContextProvider } from '@store/user/user-context';
 import { SearchMemberProvider } from '@store/search-members/searchMembers-context';
```

Comment:
can we sort the imports here somewhat like in the format where
the library imports are first
then components
then CSS followed by assets

or should it go inside another PR?

Reply:
I think this can be done in another PR